recipes/libbasisu/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class LibBasisUniversalConan(ConanFile):
+    name = "libbasisu"
+    description = "Basis Universal Supercompressed GPU Texture Codec"
+    homepage = "https://github.com/BinomialLLC/basis_universal"
+    topics = ("conan", "basis", "textures", "compression")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "sse": [True, False],
+        "x64": [True, False],
+        "zstd": [True, False],
+        "no_iterator_debug_level": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "sse": True,
+        "x64": True,
+        "zstd": True,
+        "no_iterator_debug_level": True
+    }
+    short_paths = True
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _minimal_compiler_version(self) -> bool:
+        version = tools.Version(self.settings.compiler.version.value)
+        return (self.settings.compiler == "Visual Studio" and version >= "15") \
+            or (self.settings.compiler == "gcc" and version >= "5.4") \
+            or (self.settings.compiler == "clang" and version >= "3.9") \
+            or (self.settings.compiler == "apple-clang" and version >= "10")
+ 
+    def configure(self):
+        if not self._minimal_compiler_version():
+            raise ConanInvalidConfiguration("{} {} does not support Visual Studio <= 14 with shared:True".format(self.name, self.version))
+
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_X64"] = self.options.x64
+        self._cmake.definitions["SSE"] = self.options.sse
+        self._cmake.definitions["ZSTD"] = self.options.zstd
+        self._cmake.definitions["BASISU_NO_ITERATOR_DEBUG_LEVEL"] = self.options.no_iterator_debug_level
```

Comment:
When you apply, https://github.com/conan-io/conan-center-index/pull/6468/files#r675570927, you need to use `self.options.get_safe("no_iterator_debug_level")`.
```suggestion
        self._cmake.definitions["BASISU_NO_ITERATOR_DEBUG_LEVEL"] = self.options.no_iterator_debug_level
```